README.md

```diff
@@ -57,3 +57,7 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
 
 To learn React, check out the [React documentation](https://reactjs.org/).
+
+## Useful links
+https://firebase.google.com/docs/firestore/quickstart#node.js 
+https://firebase.google.com/docs/web/setup 
```

Comment:
TY! Would be awesome to do a PR specifically to update the README.md. Could have an entire section dedicated to Firestore DB